pkg/sectigo/sectigo_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
-	. "github.com/trisacrypto/directory/pkg/sectigo"
+	"github.com/trisacrypto/directory/pkg/sectigo"
```

Comment:
Didn't like importing everything from `sectigo` into the current namespace? Was it causing an issue?

Reply:
I don't remember why I made that change but I just reverted it since it makes the code a bit more readable and doesn't appear to break anything.